webapp-django/crashstats/crashstats/views.py

```diff
@@ -1425,6 +1425,18 @@ def handle_middleware_404(crash_id, error_code):
         ]
     context['raw_keys'].sort(key=unicode.lower)
 
+    context['install_time'] = None
```

Comment:
So, the old behavior was different. It used to be that if the InstallTime is something that is clearly NOT a valid timestamp, the whole `<th>Install Time</th>...` row was removed/hidden. That was something we wrote code for because it happened. I.e. `"InstallTime": "junk",` in the raw crash. 

Now that I think about it, perhaps we just display this, in the rendered HTML, if the `InstallTime` is corrupted:
```html
<tr>
  <th>InstallTime</th>
  <td></td>
</tr>
```
What do you think?